packages/emu/src/arm7tdmi.rs

```diff
@@ -102,10 +106,14 @@ impl Cpu for Arm7tdmi {
             self.execute(op_code)
         }
     }
+
+    fn registers(&self) -> Vec<u32> {
```

Comment:
Where is used?

Reply:
It can be used from some UI widgets: in this case you can see it in cpu_inspector.rs